### ISSUE TITLE
Does it need Port forwarding in create session as listing server ?

### ISSUE THREAD

**Issue by nasriachref** [RyroNZ/UE4MasterServer#13]
hello 
i'm sorry i'm still a beginner just want to understand the system here.
i saw in your video testing you open up port forwarding is it necessary ? and if it's not can it make it so the clients host thier own game as a listing server ?

**Comment by Prazon** [comment]
Yes, port forwarding can only be avoided with NAT Punchthrough which is not included in this plugin AFAIK
